doc/source/serve/tutorials/gradio-integration.md

```diff
@@ -0,0 +1,122 @@
+# Scaling your Gradio app with Ray Serve
+
+In this guide, we will show you how to scale up your Gradio application using Ray Serve. There is no need to change the internal architecture of your Gradio app; instead, we will neatly wrap it with Ray Serve and then scale it up to access more resources.
+
+## Dependencies
+
+To follow this tutorial, you will need Ray Serve and Gradio. If you haven't already, install them by running:
+```console
+$ pip install "ray[serve]"
+$ pip install gradio
+```
+For the purposes of this tutorial, we will be working with Gradio apps that run text summarization and text generation models. **Note that you can substitute this Gradio app for any Gradio app of your own!**
+
+We will be using HuggingFace's pipeline to access the model. First, let's install the transformers module.
+```console
+$ pip install transformers
+```
+
+## Basic Example: Deploy your Gradio app with Ray Serve
+
+This example will show you an easy, straightforward way to deploy your app onto Ray Serve. Start by creating a new Python file named `demo.py` and import `GradioServer` from Ray Serve for deploying your Gradio app, `gradio`, and `transformers.pipeline` for loading text summarization models.
+```{literalinclude} ../../../../python/ray/serve/examples/doc/gradio-integration.py
+:start-after: __doc_import_begin__
+:end-before: __doc_import_end__
+```
+
+Then, we construct the (optional) Gradio app `io`:
+> **Note**: Remember you can substitute this with your own Gradio app if you want to try scaling up your own Gradio app!
```

Comment:
For all the Notes, can we maintain consistency with the other notes in the docs using this syntax:

```
:::{note}
Remember you can substitute this with your own Gradio app if you want to try scaling up your own Gradio app!
:::
```